src/XMLimport.cpp

```diff
@@ -568,883 +498,676 @@ void XMLimport::readRoom( QMultiHash<int, int> & areamRoomMultiHash, unsigned in
+            else if( name() == "mLightGreen2" ) {
+                pHost->mLightGreen_2.setNamedColor( readElementText() );
             }
-            else if( name() == "mLightYellow2")
-            {
-                pT->mLightYellow_2.setNamedColor( readElementText() );
-                continue;
+            else if( name() == "mYellow2" ) {
+                pHost->mYellow_2.setNamedColor( readElementText() );
             }
-            else if( name() == "mCyan2")
-            {
-                pT->mCyan_2.setNamedColor( readElementText() );
-                continue;
+            else if( name() == "mLightYellow2" ) {
+                pHost->mLightYellow_2.setNamedColor( readElementText() );
             }
-            else if( name() == "mLightCyan2")
-            {
-                pT->mLightCyan_2.setNamedColor( readElementText() );
-                continue;
+            else if( name() == "mCyan2" ) {
+                pHost->mCyan_2.setNamedColor( readElementText() );
             }
-            else if( name() == "mMagenta2")
-            {
-                pT->mMagenta_2.setNamedColor( readElementText() );
-                continue;
+            else if( name() == "mLightCyan2" ) {
+                pHost->mLightCyan_2.setNamedColor( readElementText() );
             }
-            else if( name() == "mLightMagenta2")
-            {
-                pT->mLightMagenta_2.setNamedColor( readElementText() );
-                continue;
+            else if( name() == "mMagenta2" ) {
+                pHost->mMagenta_2.setNamedColor( readElementText() );
             }
-            else if( name() == "mWhite2")
-            {
-                pT->mWhite_2.setNamedColor( readElementText() );
-                continue;
+            else if( name() == "mLightMagenta2" ) {
+                pHost->mLightMagenta_2.setNamedColor( readElementText() );
             }
-            else if( name() == "mLightWhite2")
-            {
-                pT->mLightWhite_2.setNamedColor( readElementText() );
-                continue;
+            else if( name() == "mWhite2" ) {
+                pHost->mWhite_2.setNamedColor( readElementText() );
             }
-            else if( name() == "mSpellDic")
-            {
-                pT->mSpellDic = readElementText();
-                continue;
+            else if( name() == "mLightWhite2" ) {
+                pHost->mLightWhite_2.setNamedColor( readElementText() );
             }
-            else if( name() == "mRoomSize" )
-            {
-                pT->mRoomSize = readElementText().toDouble();
+            else if( name() == "mSpellDic" ) {
+                pHost->mSpellDic = readElementText();
             }
-            else if( name() == "mLineSize" )
-            {
-                pT->mLineSize = readElementText().toDouble();
+            else if( name() == "mLineSize" || name() == "mRoomSize" ) {
+                // These two have been dropped from the Xml format as these are
```

Comment:
This comment is a bit unclear - mLineSize and mRoomSize are still saved, just elsewhere, aren't they? (I've tested and they are)